packages/consumption/test/modules/requests/OutgoingRequestsController.test.ts

```diff
@@ -726,12 +746,13 @@ describe("OutgoingRequestsController", function () {
 
         test("moves the Request to status 'Expired' when expiredAt is reached", async function () {
             await Given.anOutgoingRequestWith({
-                status: LocalRequestStatus.Draft,
+                status: LocalRequestStatus.Expired,
```

Comment:
```suggestion
                status: LocalRequestStatus.Draft,
```
If the Request was already expired, there'd be no reason for this test.